core/src/main/scala/org/apache/spark/ui/UIUtils.scala

```diff
@@ -363,7 +363,8 @@ private[spark] object UIUtils extends Logging {
       skipped: Int,
       reasonToNumKilled: Map[String, Int],
       total: Int): Seq[Node] = {
-    val completeWidth = "width: %s%%".format((completed.toDouble/total)*100)
+    val ratio = if (total == completed) 100.0 else (completed.toDouble/total)*100
```

Comment:
Isn't this already 100 when completed == total?

Reply:
Oh right I get it , it matters when completed is 0. I guess that could be the check, even, for clarity

Reply:
Yes, I agree with you.